internal/middleware/logging.go

```diff
@@ -2,66 +2,129 @@ package middleware
 
 import (
 	"context"
-	"time"
+	"encoding/json"
 
 	"github.com/openfga/openfga/pkg/logger"
 	serverErrors "github.com/openfga/openfga/pkg/server/errors"
+	openfgapb "go.buf.build/openfga/go/openfga/api/openfga/v1"
+	"go.opentelemetry.io/otel/trace"
 	"go.uber.org/zap"
 	"google.golang.org/grpc"
+	"google.golang.org/grpc/status"
+)
+
+const (
+	grpcServiceKey   = "grpc_service"
+	grpcMethodKey    = "grpc_method"
+	grpcTypeKey      = "grpc_type"
+	grpcCodeKey      = "grpc_code"
+	requestIDKey     = "request_id"
+	traceIDKey       = "trace_id"
+	rawRequestKey    = "raw_request"
+	rawResponseKey   = "raw_response"
+	internalErrorKey = "internal_error"
+	grpcCompleteKey  = "grpc_complete"
```

Comment:
`grpc_request_complete` ?? `grpc_complete` is pretty vague. Does it mean the server has completed start up or does it mean request complete or does it mean some other grpc component has finalized it's initialization etc..?